### PR TITLE
docs: update intro

### PR DESCRIPTION
**Pull request by simboonlong**
- https://www.pivotaltracker.com/story/show/185979911
- contract cost will update in separate pr.

### CODE REVIEW

File: docs/introduction/issuer-method-dns-did.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Using this issuer method enables you to not own a domain name. The downside is that your issuer will reflect as your public etheruem address, which can be cryptic to most common users.
+
+### DNS-DID
+
+This method concept is the same as `DNS-TXT` method. So we tie our etheruem wallet address to a custom txt record that we created in a domain name we own. Refer to below on how to implement.
+
+## How to create DNS-DID Record
+
+Make sure you have a wallet, we are only interested in the wallet address. If you do not have an existing wallet, create 1 by:
```
Comment (cavacado) [inline docs/introduction/issuer-method-dns-did.md:56]: 1 => one

File: docs/introduction/tradetrust-document-schema.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+---
+id: tradetrust-document-schema
+title: TradeTrust document schema
+sidebar_label: TradeTrust document schema
+---
+
+To explain on document schema, we will use an sample document. For this example, an invoice document scenario.
```
Comment (cavacado) [inline docs/introduction/tradetrust-document-schema.md:7]: an => a

File: docs/introduction/tradetrust-document-schema.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## Document schema
+
+Key fields to note:
+
+- $template
+- issuers
+- network
+
+> The remaining data values is dependant on the type of document you want to render. They will differ accordingly.
```
Comment (cavacado) [inline docs/introduction/tradetrust-document-schema.md:91]: is => are

**Review by superical — APPROVED** [review]

File: docs/introduction/what-is-tradetrust.md [hunk 4]
```diff
@@ -4,11 +4,25 @@ title: What is TradeTrust?
 sidebar_label: What is TradeTrust
 ---
 
-TradeTrust is a set of Governance & Legal Frameworks, Standards and future-ready Digital Infrastructure, all of which facilitates the interoperability of electronic trade documents exchanged between different digital ecosystems.
+TradeTrust is a framework that comprises globally-accepted standards that connect governments and businesses to a public blockchain to enable trusted interoperability of electronic trade documents across digital platforms AND it is offered as a digital utility.
```
Comment (cavacado) [inline docs/introduction/what-is-tradetrust.md:7]: the AND with caps is requested by the client?

**Review by cavacado — CHANGES_REQUESTED** [review]
mostly grammatical. otherwise looks fine

File: docs/introduction/what-is-tradetrust.md (hunk 4 above)
Comment (simboonlong) [reply docs/introduction/what-is-tradetrust.md:7]: as per written in the pivotal

**Review by cavacado — APPROVED** [review]

**Pull request merged**